execution-engine-commons/src/main/java/com/odysseusinc/arachne/execution_engine_common/api/v1/dto/AnalysisSyncRequestDTO.java

```diff
@@ -0,0 +1,73 @@
+package com.odysseusinc.arachne.execution_engine_common.api.v1.dto;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+public class AnalysisSyncRequestDTO {
```

Comment:
Is there any approach to use this class directly? If this is just a base class why not to make it `abstract`?

Reply:
Why would I otherwise extract it?
https://github.com/OHDSI/ArachneExecutionEngine/pull/202/files#diff-37e00a5f597e37ff3c5da899eeb8b460